packages/backend/src/plugins/permission.ts

```diff
@@ -13,8 +13,12 @@ import { PluginEnvironment } from '../types';
 
 class TestPermissionPolicy implements PermissionPolicy {
   async handle(request: PolicyQuery): Promise<PolicyDecision> {
-    if (request.permission.name === 'catalog.entity.read') {
-      return { result: AuthorizeResult.ALLOW }
+    const permissions: Array<string> = [
```

Comment:
The type should be automatically inferred here, isn't it? I don't think you need to specify it.

Reply:
I can change this, it looks cleaner without the type anyways.

Reply:
Fixed in the latest commit.